corehq/apps/linked_domain/applications.py

```diff
@@ -15,11 +15,15 @@
 )
 
 
-def get_master_app_briefs(domain_link):
+def get_master_app_briefs(domain_link, progenitor_app_id=None):
     if domain_link.is_remote:
         apps = get_brief_apps(domain_link.master_domain, domain_link.remote_details)
     else:
         apps = get_brief_apps_in_domain(domain_link.master_domain, include_remote=False)
+
+    if progenitor_app_id:
```

Comment:
is `get_master_app_briefs` also used in a context where `progenitor_app_id` should not be included?

Reply:
It is not; I'll make the parameter required.